GVRf/Framework/framework/src/main/java/org/gearvrf/GVRJassimpAdapter.java

```diff
@@ -794,14 +805,13 @@ private void loadTexture(GVRAssetLoader.AssetRequest assetRequest,
         }
         if (texIndex > 1)
         {
-            assetRequest.onTextureError(mContext, "Layering only supported for two textures, ignoring " + mFileName, mFileName);
+            assetRequest.onModelError(mContext, "Layering only supported for two textures, ignoring " + texFileName, mFileName);
             return;
         }
         if (texIndex > 0)
         {
             if (usingPBR)
             {
-                assetRequest.onTextureError(mContext, textureKey + " duplicate ignored for PBR renderer" + mFileName, mFileName);
```

Comment:
I thought this was required for PBR. Is it not now?

Reply:
You were right and I was wrong. I could take it out without disrupting the count. Down side is there is no error message.